lasagne/tests/layers/test_conv.py

```diff
@@ -181,20 +185,40 @@ def _convert(input, kernel, output, kwargs):
                                                        'stride': stride,
                                                        'flip_filters': True})
 
+    # extend (w/ and w/out symbolic output shape)
+    for symbolic in [False, True]:
+        input_shape = (4, 3, 7, 9)
+        input = np.random.random(input_shape)
+        kernel = np.random.random((16, 3, 2, 3))
+        stride = np.random.randint(2, 3, 2)
```

Comment:
This is equivalent to `stride = np.array((2, 2))` (check the documentation). I'd prefer a fixed setting anyway. We don't want the test suite's ability to spot problems be dependent on the random seed. (Technically we are already depending on the random seed for the numbers, but that's probably not as likely to expose a bug than changing parameters.)